File: abc/abc205/abc205_c.cs
// https://atcoder.jp/contests/abc205/submissions/23917252

using System;
using System.Linq;

class Program
{
    static void Main()
    {
        var input = Console.ReadLine().Split().Select(int.Parse).ToArray();
        var A = input[0];
        var B = input[1];
        var C = input[2] % 2 == 0 ? 2 : 1;

        var result = string.Empty;
        if (Math.Pow(A, C) == Math.Pow(B, C))
        {
            result = "=";
        }
        else if (Math.Pow(A, C) > Math.Pow(B, C))
        {
            result = ">";
        }
        else
        {
            result = "<";
        }
        Console.WriteLine(result);
    }
}

File: abc/abc205/abc205_c.js
// https://atcoder.jp/contests/abc205/submissions/23935030

function main(input) {
  const [A, B, C] = input.split(" ").map(Number);
  const vC = C % 2 === 0 ? 2 : 1;

  let result;
  if (Math.pow(A, vC) === Math.pow(B, vC)) {
    result = "=";
  } else if (Math.pow(A, vC) > Math.pow(B, vC)) {
    result = ">";
  } else {
    result = "<";
  }
  console.log(result);
}

main(require("fs").readFileSync("/dev/stdin", "utf8"));
